data/human/culture conversations.txt

```diff
@@ -163,6 +178,21 @@ mission "Class Trip to Port"
 
 
 
+mission "Old Defense Turrets"
+	minor
+	source
+		attributes "south"
+		not attributes "station" "frontier"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`With an annoyingly long stopover having encouraged you to do a little exploring, you pull the rental car into the designated return parking lot. The unexpected sound of large hydraulic motors draws your attention and you watch as a massive turret lifts up from behind a nearby barricade and turns toward the sky, its single cannon poised to fire at seemingly nothing.`
+			`	Looking around for some form of explanation you find a plaque placed conveniently nearby dedicating the turret, and five others like it, offering protection and discouraging piracy until relatively recent years. You can't help but wonder how well the turrets would do if they were to be actually used again.`
```

Comment:
Took a jab at replacing "dedicating".
```suggestion
			`	Looking around for some form of explanation you find a plaque placed conveniently nearby celebrating the turret, and five others like it, for offering protection and discouraging piracy until relatively recent years. You can't help but wonder how well the turrets would do if they were to be actually used again.`
```

Reply:
Commemorating, perhaps?

Reply:
Dedication plaques are applied when things are built, commemorative and celebratory plaques are applied when things are decommissioned.

Reply:
Roger. Dedication works, then.

Reply:
It works for me as well.